crates/hir_def/src/body/lower.rs

```diff
@@ -923,7 +923,10 @@ impl From<ast::LiteralKind> for Literal {
                 let text = bs.value().map(Box::from).unwrap_or_else(Default::default);
                 Literal::ByteString(text)
             }
-            LiteralKind::String(_) => Literal::String(Default::default()),
+            LiteralKind::String(str) => {
+                let text = str.value().map(Box::from).unwrap_or_else(Default::default);
+                Literal::String(text)
+            }
             LiteralKind::Byte => Literal::Uint(Default::default(), Some(BuiltinUint::U8)),
```

Comment:
In L905, there's a FIXME item:
```
// FIXME: these should have actual values filled in, but unsure on perf impact
```

1) What's a good way to measure the performance impact? I believe it's mostly changes in memory consumption, but I can only find metrics on computational performance/speed (via `integrated_benchmarks::integrated_highlighting_benchmark`)
2) Should we apply similar changes to the rest of the `LiteralKind` enums (i.e. `Byte` and `Char`)

Reply:
`$ rust-analyzer analysis-stats --memory-usage .` might work.

Reply:
2. That sounds reasonable, and we might need them for const-eval at some point.